src/commcare_cloud/ansible/roles/formplayer/filter_plugins/filters.py

```diff
@@ -0,0 +1,14 @@
+from os.path import basename
+
+
+class FilterModule:
+
+    def filters(self):
+        return {'release_names': self.release_names}
+
+    def release_names(self, hostvars, play_hosts):
```

Comment:
A docstring would be useful. Why do you need to do a set intersection? Is it so that you only get 'global' releases and not 'limited releases?

Reply:
f403d6d1ddee568ce2e7179331680e594b2b05aa

Yes, the intersection is used to exclude releases that are not present on all hosts. It does not seem useful to roll back to a release that only present on a subset of hosts.